ipcalc-geoip.c

```diff
@@ -27,6 +27,9 @@
 #include <netinet/in.h>
 #include <arpa/inet.h>
 #include <stdarg.h>
+#include <error.h>
```

Comment:
These are glibc functions not intended for portable programs.  They are nice, but I don't see a significant improvement to justify losing the portability.